### PR TITLE
topics proposed by tt-nwpmd

### PR DESCRIPTION
**Pull request by amilan17**
#44 
@yhonda21 @sebvi @tomkralidis @wmo-im/tt-wismd

### CODE REVIEW

**Comment by tomkralidis** [comment]
TT-WISMD 2024-02-12:
- @jsieland to provide review for 2024-02-20

File: topic-hierarchy/earth-system-discipline/weather/prediction/analysis/nowcasting/deterministic/index.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/analysis/nowcasting/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/analysis/seasonal/deterministic/index.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/analysis/seasonal/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/analysis/short-range/deterministic/index.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/analysis/short-range/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/forecast/nowcasting/deterministic/index.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/forecast/nowcasting/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/forecast/seasonal/deterministic/index.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/forecast/seasonal/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/forecast/short-range/deterministic/index.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/forecast/short-range/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/forecast/subseasonal/deterministic/index.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/forecast/subseasonal/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/hindcast/seasonal/deterministic/index.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/hindcast/seasonal/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/hindcast/short-range/deterministic/index.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/hindcast/short-range/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/reanalysis/seasonal/deterministic/index.csv [hunk 10]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/reanalysis/seasonal/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/reanalysis/short-range/deterministic/index.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/reanalysis/short-range/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

File: topic-hierarchy/earth-system-discipline/weather/prediction/reanalysis/subseasonal/deterministic/index.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+Name,Description,Child
```
Comment (jsieland) [inline topic-hierarchy/earth-system-discipline/weather/prediction/reanalysis/subseasonal/deterministic/index.csv:1]: Is "child" a remant or does this still have a purpose?

**Review by amilan17 — COMMENTED** [review]
remove child columns

**Comment by amilan17** [comment]
@jsieland thank you for catching that! I updated the branch -- will you please review again?

**Review by jsieland — APPROVED** [review]

**Comment by tomkralidis** [comment]
Thanks @jsieland / @amilan17 for the PR/review!

@sebvi as chair of TT-NWPMD, can you review/comment/approve if applicable?  Once we have TT-NWPMD signoff I'll do a final sweep at which point we can merge this very important enhancement to WTH.  Thanks.

File: topic-hierarchy/earth-system-discipline/weather/index.csv [hunk 13]
```diff
@@ -1,4 +1,5 @@
 Name,Description
 aviation,Aviation
+prediction,"Data sets produced by quantitative algorithms, such as numerical or statistical prediction models, describing the past, present and future meteorological states."
```
Comment (amilan17) [inline topic-hierarchy/earth-system-discipline/weather/index.csv:3]:
```suggestion
prediction,"Data sets produced by quantitative algorithms, such as numerical or statistical prediction models, describing the past, present and future meteorological states"
```
```suggestion
prediction,"Data sets produced by quantitative algorithms, such as numerical or statistical prediction models, describing the past, present and future meteorological states"
```

**Comment by sebvi** [comment]
thanks @jsieland , @amilan17 and @tomkralidis . 

I've reviewed the files in this pull request and as far as I can tell it is in full agreement with what TT-NWPMD proposed so you have my green light.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**